A3A/addons/core/functions/Missions/fn_LOG_Salvage.sqf

```diff
@@ -147,3 +187,8 @@ deleteVehicle _ship;
 
 [_vehCrewGroup] spawn A3A_fnc_groupDespawner;
 [_veh] spawn A3A_fnc_vehDespawner;
+if ("vanilla" in A3A_factionEquipFlags) then {
+	[_SDVcrewGroup] spawn A3A_fnc_groupDespawner;
+	[_diversGroup] spawn A3A_fnc_groupDespawner;
+	[_veh] spawn A3A_fnc_vehDespawner;
+};
```

Comment:
We also shouldn't assume that these groups were ever made just because the "vanilla" flag is set

Also is the `[_veh] spawn A3A_fnc_vehDespawner;` duplication necessary?